apps/api/src/services/dash.ts

```diff
@@ -0,0 +1,58 @@
+import { AppError, ErrorCodes } from '@api/utils/errors';
+import { getCourseStats, getDashOrgStats, getRecentEnrollments } from '@cio/db/queries/dash';
+import { getOrgIdBySiteName } from '@db/queries';
+
+export async function getOrganisationAnalytics(orgId?: string, siteName?: string) {
```

Comment:
Please create the type that this returns and add it here.

it would make the typeserver work faster rather than inferring the return type.